Tasks/SqlAzureDacpacDeployment/Tests/L0UtilityIPRange.ps1

```diff
@@ -19,8 +19,6 @@ $err | Add-Member -MemberType NoteProperty -Name "Exception" -Value $Exception
 $errors = @()
 $errors += $err
 
-$sqlCmd1 = New-Object PsObject
-$sqlCmd1 | Add-Member -MemberType NoteProperty -Name "Path" -Value "SqlCmd.exe"
 Register-Mock Get-Command { Write-Output $sqlCmd1 } -ParametersEvaluator { $Name -eq "SqlCmd.exe" -and $ErrorAction -eq 'SilentlyContinue' }
```

Comment:
Remove this one as well

Register-Mock Get-Command { Write-Output $sqlCmd1 } -ParametersEvaluator { $Name -eq "SqlCmd.exe" -and $ErrorAction -eq 'SilentlyContinue' }